docs/use_cases/advanced_retrieval_augmented_generation.md

```diff
@@ -22,17 +22,19 @@ In this tutorial, we will cover the following components, including code example
 3. Post-retrieval
 - Reranking
 
-### Set up
+Let's get started.
 
-Before we start diving into advanced RAG and pre-retrieval, let's set up everything we will need for this tutorial. Note that if you're on Windows, you might run into trouble setting up [bitsandbytes](https://github.com/TimDettmers/bitsandbytes), as there is no Windows support yet.
+## Set up
+
+Before we dive into advanced RAG and pre-retrieval, let's **set up** everything we'll need for this tutorial. 
+
+We'll build a [Haystack](https://docs.haystack.deepset.ai/docs/intro) pipeline using open source (sentence-transformers) embeddings and models from Huggingface. In addition, we'll need "accelerate" and "bitsandbytes" libraries to load our generative large language model (LLM) in 4-bit. This set up will enable us to run our RAG system efficiently. Indeed, this tutorial is optimized to work within free [Google Colab](https://colab.research.google.com/) GPU environments. (Note: if you're on Windows, you may run into trouble setting up [bitsandbytes](https://github.com/TimDettmers/bitsandbytes), as there is no Windows support yet.)
 
 ```bash
 pip install haystack-ai sentence-transformers accelerate -i <https://pypi.org/simple/> bitsandbytes

```

Comment:
@xaiguy
copilot marked an error in this code, if you are actually installing bitsandbytes, should it come before "-i"?

current: pip install haystack-ai sentence-transformers accelerate -i https://pypi.org/simple/ bitsandbytes
alternate suggestion: pip install haystack-ai sentence-transformers accelerate bitsandbytes -i https://pypi.org/simple/

Reply:
I think I copied that from the accelerate docs, I'll just change the position of accelerate and bitsandbytes.